templates/email/surrey/_email_bottom.html

```diff
@@ -0,0 +1,30 @@
+          </tr>
+        </table>
+      </th>
+      <th class="spacer-cell"></th>
+    </tr>
+  </table>
+  <table [% wrapper_table | safe %] style="[% wrapper_style %]">
+    <tr>
+      <th class="spacer-cell"></th>
+      <th width="[% wrapper_max_width %]" style="[% td_style %][% hint_style %]" class="hint">
+        [%~ IF email_footer.defined %]
+          [% email_footer | safe %]
+        [%~ ELSE %]
+          [% loc('This email was sent automatically, from an unmonitored email account. Please do not reply to it.') %]
+        [%~ END %]
+      </th>
+      <th class="spacer-cell"></th>
+    </tr>
+    <tr>
+      <th class="spacer-cell"></th>
+      <th width="[% wrapper_max_width %]" style="[% td_style %][% hint_style %][% banner_style %]">
+        <img src="/cobrands/surrey/images/email.png" alt="" role="presentation" aria-hidden="true" width="24px" height="24px" style="[% banner_image_style %]">
```

Comment:
This needs wrapping in `inline_image` otherwise it doesn't appear in the email client:

```suggestion
        <img src="[% inline_image('web/cobrands/surrey/images/email.png') %]" alt="" role="presentation" aria-hidden="true" width="24px" height="24px" style="[% banner_image_style %]">
```